### PR TITLE
modified git command in Travis CI script

### PR DESCRIPTION
**Pull request by ankushbhardwxj**
- Modified `git log -m -1 --name-only --pretty="format:"` to `git diff --name-only HEAD HEAD~1`
- Added a few echo commands for debugging

### CODE REVIEW

**Comment by ankushbhardwxj** [comment]
@ben-albrecht let's merge this one and do another test PR ?

File: util/travisScript.bash [hunk 1]
```diff
@@ -24,10 +24,12 @@ makeCheckAndMason () {
 # Parses the last merge commit, getting the most recent package added to the registry
 checkPackage () {
   cd ..
-  package=$(git log -m -1 --name-only --pretty="format:")
+  package=$(git diff --name-only HEAD HEAD~1)
   end=".end"
   path="$package$end"
   cd $(dirname $path)
+  echo $package
```
Comment (ben-albrecht) [inline util/travisScript.bash]: Mind adding a prefix to these for improved readability?

e.g.

```sh
echo "package detected from git diff: ${package}"
echo "package path: ${path}"
```

Comment (ankushbhardwxj) [reply util/travisScript.bash]: @ben-albrecht Done 👍

**Review by ben-albrecht — APPROVED** [review]

**Comment by ankushbhardwxj** [comment]
@ben-albrecht Sorry made a mistake. Rectifying in new PR.

**Pull request merged**